assets/shiny-server.js

```diff
@@ -93,14 +93,12 @@
           } catch (e) {
             return false;
           }
-        }
+        };
 
-        var availableOptions = ['websocket', 'xdr-streaming', 'xhr-streaming', 
-            'iframe-eventsource', 'iframe-htmlfile', 'xdr-polling', 
-            'xhr-polling', 'iframe-xhr-polling', 'jsonp-polling'];
+        var availableOptions = {{{protocols}}}; 
```

Comment:
I couldn't easily find exactly what our version of Handlebars supported and I didn't want to get bitten if/when we update. This felt safer to me.